static/css/main.css

```diff
@@ -44,6 +44,11 @@ ul {
   width: 40px;
 }
 
+.twitter > iframe {
```

Comment:
Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

*Origin: CSSLintBear, Section: `css`.*